content/docs/2.6/scalers/activemq.md

```diff
@@ -0,0 +1,87 @@
++++
+title = "ActiveMQ"
+availability = "v2.6+"
+maintainer = "Community"
+description = "Scale applications based on ActiveMQ Queue."
+layout = "scaler"
+go_file = "activemq_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `activemq` trigger that scales based on a ActiveMQ Queue.
+
+```yaml
+triggers:
+  - type: activemq
+    metadata:
+      managementEndpoint: "activemq.activemq-test:8161"
+      destinationName: "testQueue"
```

Comment:
Would it make sense to just call this `queueName`?

Reply:
It defined as destination name in Rest API template. I also thought to change but I wasn't sure that it'll make confusion or not so I kept as destination name. Do you suggest change it as queueName?

Reply:
It being mentioned as destination in ActiveMQ documents. For instance: https://activemq.apache.org/how-do-i-create-new-destinations

Reply:
This is the Rest API template that I monitor message count in scaler. It use as destinationName in the template: "http://<<managementEndpoint>>/api/jolokia/read/org.apache.activemq:type=Broker,brokerName=<<brokerName>>,destinationType=Queue,destinationName=<<destinationName>>/QueueSize". 

This document explains object names  more clear under the ActiveMQ MBeans Reference title: https://activemq.apache.org/jmx

Reply:
Great, I'd keep this aligned with ActiveMQ